src/components/TodoItem.tsx

```diff
@@ -1,23 +1,35 @@
-import { ChangeEvent } from "react";
+import { ChangeEvent, MouseEvent } from "react";
 import { TodoItemType } from "../App";
 
 type TodoItemProps = {
     index: number;
     item: TodoItemType;
-    handleTodoItemChecked: (index: number, newCheckedState: boolean) => void;
+    onTodoItemChecked: (index: number, newCheckedState: boolean) => void;
+    onDeleteClick: (index: number) => void;
 }
 
-export function TodoItem({ index, item, handleTodoItemChecked }: TodoItemProps) {
+export function TodoItem({ index, item, onTodoItemChecked, onDeleteClick }: TodoItemProps) {
     const { isDone, name } = item;
 
     const handleOnChange = (event: ChangeEvent<HTMLInputElement>) => {
       const newCheckedState = event.target.checked;
 
-      handleTodoItemChecked(index, newCheckedState);
+      onTodoItemChecked(index, newCheckedState);
     };
 
-    const handleDeleteClick = () => {
-      // TODO: build me!
+    function handleDeleteClick() {
+      onDeleteClick(index);
+    }
+
+    if (isDone){
+      return (
+        <div style={{
+          marginBottom: `2px`
+        }}>
+          <input type="checkbox" checked={isDone} onChange={handleOnChange}/>
+          <p><s>{name}</s></p>
+          <button onClick={handleDeleteClick}>Delete</button>
+        </div>)
```

Comment:
The parenthesis on line 32 here should be on a new line with a semi-colon.

Replace
```
    </div>)
```
with
```
    </div>
);
```

Reply:
handled in [77230de](https://github.com/ZachWitzig/todoist/pull/3/commits/77230de0aa48039c04a2d2707327e22d807c5848)